bitnami/jaeger/templates/_helpers.tpl

```diff
@@ -147,7 +147,7 @@ an issue about env vars collision with the chart name when the release name is s
 ref. https://github.com/jaegertracing/jaeger-operator/issues/1158
```

Comment:
Could you check this comment? Could you run a test with [HotROD](https://github.com/jaegertracing/jaeger/tree/main/examples/hotrod) or any other application?, if everything is working as expected now, please remove or amend this comment.